lib/commands/element.js

```diff
@@ -246,6 +252,75 @@ commands.clear = async function (el) {
   await retry(5, this.proxyCommand.bind(this), `/element/${el}/clear`, 'POST');
 };
 
+commands.getContentSize = async function (el) {
+  if (this.isWebContext()) {
+    throw new errors.NotYetImplementedError('Support for getContentSize for webcontext is not yet implemented. Please contact an Appium dev');
+  }
+
+  const type = await this.getAttribute('type', el);
+
+  if (type !== "XCUIElementTypeTable" &&
+      type !== "XCUIElementTypeCollectionView") {
+    throw new Error(`Can't get content size for type '${type}', only for ` +
+                    `tables and collection views`);
+  }
+  let locator = "*";
+  if (type === "XCUIElementTypeTable") {
+    // only find table cells, not just any children
+    locator = "XCUIElementTypeCell";
+  }
+
+  let contentHeight = 0;
+  let children = await this.findElOrEls(`class chain`, locator, true, el);
+  if (children.length === 1) {
+    // if we know there's only one element, we can optimize to make just one
+    // call to WDA
+    const rect = await this.getRect(_.first(children));
+    contentHeight = rect.height;
+  } else if (children.length) {
+    // otherwise if we have multiple elements, logic differs based on element
+    // type
+    switch (type) {
+      case "XCUIElementTypeTable": {
+        const firstRect = await this.getRect(_.first(children));
```

Comment:
I assume it's `_.head` in the recent lodash

Reply:
yep, `_.first` still works as an alias but i'll change it